nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/test/java/org/nuxeo/ecm/automation/core/operations/coldstorage/RequestRetrievalFromColdStorageTest.java

```diff
@@ -52,6 +58,12 @@
     @Inject
     protected AutomationService automationService;
```

Comment:
Unused, should we removed along with https://github.com/nuxeo/nuxeo/pull/3666/commits/938191c373574825b97c76659cc5c57026ab8865